### PR TITLE
Copy-edit pass

### PR DESCRIPTION
**Pull request by shavidissa**
Updated minor issues in the .md file.
- Introduced working groups in the intro (the abbreviation was not introduced previously).
- Removed extra words (that).
- Added a missing comma.

### CODE REVIEW

File: generator/README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # SIG Doc builder
 
 This folder contains scripts to automatically generate documentation about the
-different Special Interest Groups (SIGs) of Kubernetes. The authoritative
+different Special Interest Groups (SIGs) and Working Groups (WGs)of Kubernetes. The authoritative
```
Comment (cblecker) [inline generator/README.md]: nit: space after (WGs)

File: generator/README.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ The documentation follows a template and uses the values from [`sigs.yaml`](/sig
 Time zones make everything complicated.
 And Daylight Saving time makes it even more complicated.
 Meetings are specified with a time zone and we generate a link to http://www.thetimezoneconverter.com/ so that people can easily convert it to their local time zone.
-To make this work you need to specify the time zone in a way that that web site recognizes.
+To make this work you need to specify the time zone in a way that web site recognizes.
```
Comment (cblecker) [inline generator/README.md]: "in a way that the website recognizes" perhaps?

**Comment by spiffxp** [comment]
/lgtm cancel
good point @cblecker

**Comment by shavidissa** [comment]
Thank you! Good catches :) Updated the pr.

File: generator/README.md [hunk 3]
```diff
@@ -17,12 +17,14 @@ The documentation follows a template and uses the values from [`sigs.yaml`](/sig
 **Time Zone gotcha**:
 Time zones make everything complicated.
 And Daylight Saving time makes it even more complicated.
-Meetings are specified with a time zone and we generate a link to http://www.thetimezoneconverter.com/ so that people can easily convert it to their local time zone.
-To make this work you need to specify the time zone in a way that that web site recognizes.
+Meetings are specified with a time zone and we generate a link to http://www.thetimezoneconverter.com/ so 
+
```
Comment (cblecker) [inline generator/README.md]: is there a reason for this space?

**Pull request merged**